application/src/main/java/org/togetherjava/tjbot/commands/moderation/temp/TemporaryBanAction.java

```diff
@@ -33,30 +32,4 @@
             @NotNull String reason) {
         return guild.unban(target).reason(reason);
     }
-
-    @Override
-    public @NotNull FailureIdentification handleRevokeFailure(@NotNull Throwable failure,
-            long targetId) {
-        if (failure instanceof ErrorResponseException errorResponseException) {
-            if (errorResponseException.getErrorResponse() == ErrorResponse.UNKNOWN_USER) {
-                logger.debug(
-                        "Attempted to revoke a temporary ban but user '{}' does not exist anymore.",
-                        targetId);
-                return FailureIdentification.KNOWN;
-            }
-
-            if (errorResponseException.getErrorResponse() == ErrorResponse.UNKNOWN_BAN) {
-                logger.debug(
-                        "Attempted to revoke a temporary ban but the user '{}' is not banned anymore.",
-                        targetId);
-                return FailureIdentification.KNOWN;
-            }
-
-            if (errorResponseException.getErrorResponse() == ErrorResponse.MISSING_PERMISSIONS) {
-                logger.warn("Attempted to revoke a temporary ban but the bot lacks permission.");
-                return FailureIdentification.KNOWN;
-            }
-        }
-        return FailureIdentification.UNKNOWN;
```

Comment:
For BAN the logic changes now, that ur using the merged logic. Is the difference in behavior acceptable or not?